jq_tags_parser.go

```diff
@@ -70,6 +72,10 @@ func (p *JQTagsParser) parse(programs []*JQFieldParser, data string) []opslevel.
 			}
 		}
 	}
+	// TODO: fix this hack to make the hashmap keys order be consistent (needed for testing.)
```

Comment:
Fixed in next PR in the chain -- basically, ordering is not guaranteed by `jq` and so our tests need to take that into account. 

more on [jq and ordering](https://stackoverflow.com/a/38210345)

Reply:
The SO link references jq 1.6 docs installed on ubuntu specifically, and it also assumes command line usage given `-S` flag. Have you looked into [sort(), sort_by() from the docs](https://jqlang.github.io/jq/manual/#sort-sort_by)?

Reply:
@davidbloss I considered this but ultimately decided it's not a great idea since not all expressions will be sortable AFAICT. It would be easier to just change this on the test end like in #34.